tests/test_workers/test_tasks/test_build.py

```diff
@@ -159,7 +166,7 @@ def test_get_local_pull_spec(request_id, arch):
     assert re.match(f'.+:{request_id}-{arch}', rv)
 
 
-@mock.patch('iib.workers.tasks.build.skopeo_inspect')
+@mock.patch('iib.workers.tasks.utils.skopeo_inspect')
```

Comment:
Could it be better to move the test of functions moved to utils.py to test_utils.py?

Reply:
Could you please fix this too?